internal/testplanet/node.go

```diff
@@ -73,9 +75,22 @@ func (node *Node) DialPointerDB(destination *Node, apikey string) (pdbclient.Cli
 		}
 		return piececlient.NewPSClient
 	*/
+
+	// TODO: handle disconnect
 	return pdbclient.NewClient(node.Identity, destination.Addr(), apikey)
 }
 
+// DialOverlay dials destination with apikey and returns pointerdb Client
+func (node *Node) DialOverlay(destination *Node) (pb.OverlayClient, error) {
```

Comment:
Can we have this method returning the `overlay.Client` interface? This is what, for example, the Segment Store expects as param

Reply:
sounds good